src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,54 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER_OF_ELEMENTS = 10;
+    private final Node<K, V>[] nodeArray = new Node[MAX_NUMBER_OF_ELEMENTS];
+    private int size = 0;
+
```

Comment:
better to initialize in constructor
int has default value 0
```suggestion
    private final Node<K, V>[] nodeArray;
    private int size;

    public StorageImpl() {
        nodeArray = new Node[MAX_NUMBER_OF_ELEMENTS];
    }
```